crates/engine/tree/src/tree/payload_processor/multiproof.rs

```diff
@@ -34,6 +34,10 @@ use std::{
 };
 use tracing::{debug, error, trace};
 
+/// Default upper bound for inflight multiproof calculations. These would be sitting in the queue
+/// waiting to
+pub const DEFAULT_MULTIPROOF_INFLIGHT_LIMIT: usize = 256;
```

Comment:
initially wanted this as a cli arg, but dont think it makes sense as its too hard to configure

Reply:
reasonable